recipes-sdk/aws-iot-device-sdk-python-v2/aws-iot-device-sdk-python-v2.inc

```diff
@@ -4,7 +4,7 @@ HOMEPAGE = "https://github.com/aws/aws-iot-device-sdk-python-v2.git"
 LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=f91e61641e7a96835dea6926a65f4702"
 
-BRANCH ?= "master"
+BRANCH ?= "main"
 
 SRC_URI = "git://github.com/aws/aws-iot-device-sdk-python-v2.git;branch=${BRANCH};name=aws-iot-device-sdk-python-v2"
 SRCREV = "${AUTOREV}"
```

Comment:
Please replace AUTOREV with fixed SRCREV as well, latest is 844bf38ebb13e316ea8100e364121e616c9e71df